package.json

```diff
@@ -35,8 +35,7 @@
     "clean": "rm -rf assets/stylesheets build dist node_modules"
   },
   "engines": {
-    "node": "^18",
-    "npm": "^9"
```

Comment:
npm v10 is the latest but we don’t hugely care; this package doesn’t do anything special